src/api/db/migrate/20200522151615_backfill_add_communication_scope_to_status_message.rb

```diff
@@ -0,0 +1,12 @@
+class BackfillAddCommunicationScopeToStatusMessage < ActiveRecord::Migration[6.0]
+  disable_ddl_transaction!
+
+  def up
+    StatusMessage.unscoped.in_batches do |relation|
+      # rubocop:disable Rails/SkipsModelValidations
+      relation.update_all communication_scope: 0
+      # rubocop:enable Rails/SkipsModelValidations
+      sleep(0.01)
```

Comment:
Hm, why sleep?

Reply:
When I tried to run a migration to add `communication_scope` with default `0`, `strong_migrations` proposed this solution to avoid downtime. It is exactly what `strong_migrations`, I only added rubocop enable/disable.

Reply:
hm okay...

Reply:
https://www.rubydoc.info/gems/strong_migrations/0.6.2